### PR TITLE
Update interested-parties.md with Graeme Hay and Erick Bourgeois from Morgan Stanley

### PR DESCRIPTION
**Pull request by grmhay**

### CODE REVIEW

**Comment by lloydchang** [comment]
@grmhay @ebourgeois  Generally re: DCO, I found the the following useful.  Thank you!

@scottrigby wrote in https://github.com/gitops-working-group/gitops-working-group/pull/117
>  just need to amend your commit to add your DCO signoff
>
> PS, you may be interested in this fun plugin I made to help with this when using the GitHub UI 😸 https://github.com/scottrigby/dco-gh-ui/
>
> I made this because this issue eventually happens to almost everyone contributing to CNCF/LF projects that require DCO.

**Comment by grmhay** [comment]
Thanks - will check that out

File: interested-parties.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Mambu | Adrian Mârza
 MayaData | Uma Mukkara
 MayaData | Karthik S
 Microsoft | Chris Sanders
+Morgan Stanley | Erick Bourgeois
+Morgan Stanley | Graeme Hay
```
Comment (lloydchang) [inline interested-parties.md:62]: Thanks @ebourgeois and @grmhay — A tip similar to https://github.com/gitops-working-group/gitops-working-group/pull/170#pullrequestreview-780057521
> Hi! Welcome! Could you add your GitHub username next to your name - we had not done this with previous interested parties and then realized we really had no efficient way to be able to reach out to all of you so I want to make sure we start fixing that ;)

**Review by lloydchang — CHANGES_REQUESTED** [review]

**Comment by scottrigby** [comment]
Hi @grmhay and @ebourgeois - would you please make a new PR adding yourself to [interested-parties.md](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/interested-parties.md) in the GitOps WG new location under the App Delivery TAG repo?
Thanks!

**Pull request closed without merging**